src/components/IdeaCard/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import { teams } from '../../server/mockServerData';
+import  './IdeaCard.css';
+
+const IdeaCard = ({
+  name,
+	description,
+	teamId,
+	id
+}) => {
+
+	const team = teams.find((team) =>  {
+		if (team.id === teamId) {
+			return team
+		}
+	});
+
+	const joinIdeaOverlay = (id) => {
+		console.log('opens join idea overlay')
+	};
+
+	const openIdeaOverlay = (id) => {
+		console.log('opens Idea overlay')
+	};
+
+	return (
+		<div className='IdeaCard' key={id}>
+			<div className="IdeaCard-Container">
+				<div className="IdeaCard-Info">
+					<div className="IdeaCard-Info-Name">
+						{name}
+					</div>
+					<div className="IdeaCard-Info-Description">
+						{description}
+					</div>
+				</div>
+				<div className="IdeaCard-Actions">
+					<div className="Team">
```

Comment:
omg is this normal formatting? it looks very weird on my screen :D

Reply:
It looks good in my editor, because I can not manage to set up eslint)))) 
I dont know why, when I use tabs it looks like that. When u set up eslint, I will fix everything

Reply:
hahah ok!